lib/nepali_calendar/fiscal_year.rb

```diff
@@ -0,0 +1,25 @@
+class NepaliCalendar::FiscalYear < NepaliCalendar::Calendar
```

Comment:
This class apparently also needs `current_fiscal_year` class method; which is currently implemented ditto-ish in two different places - bs_calendar.rb and ad_calendar.rb. Remove the existing implementations.